src/main/java/org/kairosdb/datastore/cassandra/CassandraClientImpl.java

```diff
@@ -81,19 +81,19 @@ public void close() {
 	}
 
     private void logConnectionPoolConfig(PoolingOptions poolOpts) {
-        logger.debug("Core connections per host (remote): " + poolOpts.getCoreConnectionsPerHost(HostDistance.REMOTE));
-        logger.debug("Core connections per host (local): " + poolOpts.getCoreConnectionsPerHost(HostDistance.LOCAL));
-        logger.debug("Max connections per host (remote): " + poolOpts.getMaxConnectionsPerHost(HostDistance.REMOTE));
-        logger.debug("Max connections per host (local): " + poolOpts.getMaxConnectionsPerHost(HostDistance.LOCAL));
-        logger.debug("Max requests per connection (remote): " + poolOpts.getMaxRequestsPerConnection(HostDistance.REMOTE));
-        logger.debug("Max requests per connection (local): " + poolOpts.getMaxRequestsPerConnection(HostDistance.LOCAL));
-        logger.debug("Max queue size: " + poolOpts.getMaxQueueSize());
-        logger.debug("Pool timeout mills: " + poolOpts.getPoolTimeoutMillis());
-        logger.debug("Idle timeout seconds: " + poolOpts.getIdleTimeoutSeconds());
+        logger.info("Core connections per host (remote): " + poolOpts.getCoreConnectionsPerHost(HostDistance.REMOTE));
```

Comment:
Are you sure that `info` is enough? Correct me if I'm wrong, but I think we have a `warn` level in the config